gdl_hyperopt_template.py

```diff
@@ -0,0 +1,152 @@
+"""Hyperparamater optimization for GDL using hyperopt
+
+This is a template for using hyperopt with GDL. The my_space variable currently needs to
+be modified here, as well as GDL config modification logic within the objective_with_args
+function.
+
+"""
+
+import argparse
+from pathlib import Path
+import pickle
```

Comment:
See https://docs.python.org/3/library/pickle.html.  And https://medium.com/swlh/object-serialization-and-deserialization-in-python-5fad3c2970a4 (Marshmallow).  Maybe even more "strategic" would be to start using Google's [protobuf](https://pypi.org/project/protobuf-serialization/).  This is how Mapbox serializes their vector tiles IIRC